CHANGELOG.md

```diff
@@ -22,7 +22,7 @@
 - Added support for override values with environment variable expansion in the `watches.yaml` file for Helm-based operators. ([#2325](https://github.com/operator-framework/operator-sdk/pull/2325))
 
 ### Changed
-
+- Changed error wrapping according to Go version 1.13+ [error handling](https://blog.golang.org/go1.13-errors). ([#2355](https://github.com/operator-framework/operator-sdk/pull/2355))
```

Comment:
Hi @nrchakradhar,

We had a new release so it needs to be updated with the master and the CHANGELOG properly updated. Could you please do that for we are able to merge?